core/src/cluster_info.rs

```diff
@@ -694,6 +710,16 @@ impl ClusterInfo {
             .collect()
     }
 
+    fn tvu_peers_and_stakes(
```

Comment:
nit: should we mention that this sorts the peers by stake ? `sorted_tvu_peers_and_stakes` or `sorted_tvu_peers_with_stakes`, same with `peers_and_stakes`

Reply:
Does it sort?

Reply:
Ah it does.